application/src/test/java/org/opentripplanner/routing/api/request/preference/StreetPreferencesTest.java

```diff
@@ -109,9 +109,14 @@ void testToString() {
       "routingTimeout: 3s, " +
       "elevator: ElevatorPreferences{boardTime: 2m}, " +
       "intersectionTraversalModel: CONSTANT, " +
-      "accessEgress: AccessEgressPreferences{penalty: TimeAndCostPenaltyForEnum{CAR_TO_PARK: " +
+      "accessEgress: AccessEgressPreferences{penalty: TimeAndCostPenaltyForEnum{" +
+      "CAR: (timePenalty: 20m + 2.0 t, costFactor: 1.50), " +
+      "CAR_TO_PARK: " +
       CAR_PENALTY +
```

Comment:
The name of this variable is a bit misleading since it's only used for `CAR_TO_PARK`.

Reply:
Fixed [review: Apply fixes from review](https://github.com/opentripplanner/OpenTripPlanner/pull/6302/commits/1e61ccd1d182bc6783b59ab5d845d3b23836ab07)